modules/base/rendering/renderableplanet.cpp

```diff
@@ -349,7 +349,7 @@ void RenderablePlanet::render(const RenderData& data) {
     glm::dmat4 rot = glm::rotate(glm::dmat4(1.0), M_PI_2, glm::dvec3(1, 0, 0));
     glm::dmat4 roty = glm::rotate(glm::dmat4(1.0), M_PI_2, glm::dvec3(0, -1, 0));
     //glm::dmat4 rotProp = glm::rotate(glm::dmat4(1.0), glm::radians(static_cast<double>(_rotation)), glm::dvec3(0, 1, 0));
-    modelTransform = modelTransform * rot /** roty*/ /** rotProp*/;
+    modelTransform = modelTransform * rot * roty /** rotProp*/;
```

Comment:
Yeah, I need to look more into it again...  the first rotation moves the earth from rotating on its belly (aka. north pole pointing at the sun) to the correct side up.  The second rotation rotates the meridian into the correct location. I don't know why, but both rotations are necessary to produce the correct locations on Earth for their local time (without the second rotation, every position is 12h off sync, basically)